EIPS/eip-2135.md

```diff
@@ -0,0 +1,74 @@
+
+---
+eip: 2135
+title: Consumable Interface
+author: Zainan Victor Zhou (@xinbenlv)
+discussions-to: https://github.com/ethereum/EIPs/issues/2135
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-06-23
+requires (*optional): N/A
+replaces (*optional): N/A
+---
+
+
+ERC: Consumable Interface
+
+## Simple Summary
+An interface marking certain digital assets being consumable. 
+
+## Abstract
+The interface identifies functions and events needed for creating a contract to be able to mark a digital asset as "consumable", and react to the request of "consumption".
+
+## Motivation
+Being a digital assets sometimes means a consumable power. A most common seen example would be a concert ticket. It will be "consumed" at the moment the ticket-holder use the ticket to get access to enter a concert. 
```

Comment:
> Being a digital assets sometimes means a consumable power.

This sentence doesn't seem to make sense.

>  A most common seen example

It's either '**the** most common example' or '**a** common example'.

Reply:
Changed to "One of the most common seen examples"